tests/test_open.py

```diff
@@ -0,0 +1,8 @@
+"""Tests of file opening"""
+
+import fiona
+
+
+def test_open_shp():
+    """Open a shapefile"""
+    assert fiona.open("tests/data/coutwildrnp.shp")
```

Comment:
This path assumes pytest has been called from the root project directory, which may not be true.